recipes/cylc-flow/meta.yaml

```
@@ -0,0 +1,70 @@
+{% set name = "cylc-flow" %}
+{% set version = "8.0a1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "36f4f4363cb5cc7a3a613d2db6e84cc41c2825c6cd011191cc4821c5ca9c81a4"
+
+build:
+  number: 0
+  skip: True  # [not linux]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+
+requirements:
+  host:
+    - ansimarkup >=1.0.0
+    - colorama >=0.4
+    - graphene ==2.1.6
+    - jinja2 >=2.10.1
+    - markupsafe >=1.1
+    - metomi-isodatetime >=1!2.0.1
+    - pip
+    - protobuf >=3.7
+    - python >=3.7
```

Python should only be constrained this way if it's a `noarch: python` package:
```suggestion
    - python
```

---

Thanks for all the feedback!, I'll act on it ASAP in a single feedback commit.

For context, we do want to constrain to versions ``3.7`` or above so it is a case of adding the ``noarch`` statement in rather than taking out the minimum version specification.

---

You can still constrain the version but you shouldn’t do it this way as Python is special.

For packages which are noarch there is one build and `python >=3.7` would be correct. In this case it can’t be noarch as there are scripts included (instead of entry_points) and instead there is one build per platform per Python version. You then have to use `skip: true` to prevent the windows, Python 2.7 and Python 3.6 builds from being ran.

---

Ah I see, thanks for the clarification. I'll follow your pointers.